### DISCUSSION TITLE
工作流定时任务自动触发不了

### DISCUSSION THREAD

**Discussion by LYSCK** [apache/dolphinscheduler#6554] [General]
在dolphinscheduler-1.3.7定义工作流定时任务创建后 ，自动触发不了执行任务不知是什么原因导致

**Comment by MarlboroBoy** [comment]
我在1.3.5上碰到了同样的问题，定时作业会有丢失的情况出现

**Comment by B1ack7** [comment]
是触发了 但是任务实例丢失吗，我也碰到了一样的问题。

**Comment by bigonege** [comment]
我的也是，不执行，command表很多数据了，就是不执行
